### DISCUSSION TITLE
https://penguins-eggs.net/blog/triple-somersault?fbclid=IwAR08wxfcIx8mIVqj_BlT-DLXYiEnZfwqtRIRAw8SwC44b_FmQowbO31j09o_aem_Acv7qhl7O2fi4rW406JC1hBy60IjVcjpRTmW1YemfHV2_-bnXbc33XAH8PcHgh6Fel8

### DISCUSSION THREAD

**Discussion by giscus** [pieroproietti/penguins-blog#14] [General]
# https://penguins-eggs.net/blog/triple-somersault?fbclid=IwAR08wxfcIx8mIVqj_BlT-DLXYiEnZfwqtRIRAw8SwC44b_FmQowbO31j09o_aem_Acv7qhl7O2fi4rW406JC1hBy60IjVcjpRTmW1YemfHV2_-bnXbc33XAH8PcHgh6Fel8

Una volta creato e testato eggs per l'architettura ARM64, abbiamo il problema di provarlo su hardware reale.

https://penguins-eggs.net/blog/triple-somersault?fbclid=IwAR08wxfcIx8mIVqj_BlT-DLXYiEnZfwqtRIRAw8SwC44b_FmQowbO31j09o_aem_Acv7qhl7O2fi4rW406JC1hBy60IjVcjpRTmW1YemfHV2_-bnXbc33XAH8PcHgh6Fel8

**Comment by fconidi** [comment]
Vero! sono un po’ lento ma ti darò un feed ;)
